src/main/resources/application.properties

```diff
@@ -1 +1,55 @@
-messages.file=messages.txt
+#================================
+# Data processing app properties
+#================================
+
+#------------------------
+# Global settings
+#------------------------
+# Directory name that will be used for the manual intervention directory
+# Created in the users' home folders
+# e.g. /home/<user1>/error
+users.error.directory.name=error
```

Comment:
NIT: I would place this below the scanner.directory

Reply:
it is a global setting and not specific for the scanner part, so I favor it to be separated :D